pkgs/kernels/linux-ioregionfd-5.15.nix

```
@@ -0,0 +1,25 @@
+{ buildLinux, fetchFromGitHub, linuxPackages_5_15, fetchurl, modDirVersionArg ? null, ... }@args:
+
+buildLinux (args // rec {
+  version = "5.15";
+  modDirVersion = if (modDirVersionArg == null) then
+    builtins.replaceStrings [ "-" ] [ ".0-" ] version
+      else
+    modDirVersionArg;
```

> Error: modDirVersion 5.15 specified in the Nix expression is wrong, it should be: 5.15.0
```suggestion
  modDirVersion = "5.15.0";
```